src/main/resources/templates/userSubjects.html

```diff
@@ -21,87 +21,112 @@
     <script src="/js/navbarFooter.js" type="text/javascript"></script>
     <script src="/js/deleteSubject.js" type="text/javascript"></script>
     <script src="/js/subjectFormModal.js"></script>
+    <script src="/js/userSubjects.js"></script>
```

Comment:
this naming is problematic. what do you do in the js? you add a grade to a usersubject right? so name it "addGrade" or something like that